docs/articles/commands/intro.md

```diff
@@ -1,221 +1,328 @@
+```cs
+[Command("greet")]
+public async Task GreetCommand(CommandContext ctx, string name)
+```
+CommandsNext will now interpret this as a command named *greet* that takes one argument.
 
+Next, replace our original response message with an [interpolated string](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated)
+which uses our new parameter.
 ```cs
-public async Task Hi(CommandContext ctx)
+public async Task GreetCommand(CommandContext ctx, string name)
 {
-
+    await ctx.RespondAsync($"Greetings, {name}! You're pretty neat!");
 }
 ```
 
-Now, put the following code inside that method: `await ctx.RespondAsync($"👋 Hi, {ctx.User.Mention}!");`
+That's all there is to it. Smack `F5` and test it out in a channel your bot account has access to.
 
-You're not ready yet. Above the method, put the Command attribute. It should look like this: `[Command("hi")]`
+![Greet Part 2: Electric Boogaloo](/images/commands_intro_04.png)
 
-Put together, the class should now look like this:
+<br/>
+Now, you may have noticed that providing more than one word simply does not work.<br/>
+For example, `!greet Luke Smith` will result in no response from your bot.
 
-```cs
-using System;
-using System.Collections.Generic;
-using System.Text;
-using System.Threading.Tasks;
-using DSharpPlus;
-using DSharpPlus.CommandsNext;
-using DSharpPlus.CommandsNext.Attributes;
+This fails because a valid [overload](#command-overloads) could not be found for your command.
 
-namespace MyFirstBot
-{
-    public class MyCommands
-    {
-        [Command("hi")]
-        public async Task Hi(CommandContext ctx)
-        {
-            await ctx.RespondAsync($"👋 Hi, {ctx.User.Mention}!");
-        }
-    }
-}
+CommandsNext will split arguments by whitespace. This means `Luke Smith` is counted as two separate arguments; `Luke` and `Smith`.
+In addition to this, CommandsNext will attempt to find and execute an overload of your command that has the *same number* of provided arguments.
+Together, this means that any additional arguments will prevent CommandsNext from finding a valid overload to execute.
+
+The simplest way to get around this would be to wrap your input with double quotes.<br/>
+CommandsNext will parse this as one argument, allowing your command to be executed.
+```
+!greet "Luke Smith"
 ```
 
-What this command will do is posting a message that contains the :wave: emoji, and says Hi, followed by a mention of the 
-user who invoked the command.
+A more obvious solution is to add additional parameters to the method signature of your command method.<br/>
+```cs
+public async Task GreetCommand(CommandContext ctx, string firstName, string lastName)
+```
+
+
+Lastly, you can use the `RemainingText` attribute on your parameter.<br/>
+This attribute will instruct CommandsNext to parse all remaining arguments into that parameter. 
+```cs
+public async Task GreetCommand(CommandContext ctx, [RemainingText]string name)
```

Comment:
Nitpick: we typically add a space after attributes placed on function arguments.